pages/price-feeds/how-pyth-works/schedule-format.mdx

```diff
@@ -0,0 +1,52 @@
+# Schedule Format
```

Comment:
I think you should put both schedules in a single page with the current one on top and the deprecated one below that.

Reply:
yeah makes sense. Done.